dependencies.yaml

```
@@ -66,12 +66,26 @@ dependencies:
               arch: x86_64
               cuda: "11.8"
             packages:
+              - cuda-version=11.8
               - nvcc_linux-64=11.8
           - matrix:
               arch: aarch64
               cuda: "11.8"
             packages:
+              - cuda-version=11.8
               - nvcc_linux-aarch64=11.8
+          - matrix:
+              arch: x86_64
+              cuda: "12.0"
+            packages:
+              - cuda-version=12.0
+              - cuda-nvcc
+          - matrix:
+              arch: aarch64
+              cuda: "12.0"
+            packages:
+              - cuda-version=12.0
+              - cuda-nvcc
```

Identical package lists for each arch can be simplified:
```suggestion
          - matrix:
              cuda: "12.0"
            packages:
              - cuda-version=12.0
              - cuda-nvcc
```